src/service/receipt.service.ts

```diff
@@ -41,41 +60,53 @@ export class ReceiptService {
     public formatTransactionStatements(statement: any): string {
         let txt = '';
         txt += chalk.green('transactionStatements:\t') + '\n';
-        txt += '-'.repeat('transactionStatements:\t'.length) + '\n\n';
         statement.transactionStatements.map((transaction: TransactionStatement, transactionIndex: number) => {
-            txt += 'height:\t\t' + transaction.height + '\n';
             transaction.receipts.map((receipt: any, receiptIndex: number) => {
-                txt += '<index: ' + transactionIndex + '-' + receiptIndex + '>\t';
                 if (receipt instanceof BalanceTransferReceipt) {
```

Comment:
I'd create one class per receipt table, so we can test each table in the future.

Example: https://github.com/nemtech/nem2-cli/blob/master/src/commands/block/header.ts#L36